mtags-shared/src/main/scala/scala/meta/internal/pc/AutoImportPosition.scala

```diff
@@ -19,18 +21,25 @@ case class AutoImportPosition(
 }
 
 object AutoImportPosition {
+  private val endOfLineCharacters = Set('\t', '\n')
```

Comment:
`\t` is for tab, it doesn't need to be at the end of the line. Did you mean `\r` ?

Reply:
Yes! I don't know how many times I've tried to get this tiny PR right today 🤯 😮‍💨

Reply:
TBH I started to even wonder if perhaps `\t` characters should contribute to the detected indentation (not really sure how and if this is handled elsewhere), but even if that would be the case, then probably it's out of scope of this PR. :sweat_smile: